### PR TITLE
examples for a REPL and custom steps with @Doc that turn up in .help texts

### PR DESCRIPTION
**Pull request by mpollmeier**

### CODE REVIEW

File: src/main/scala/org/codeminers/standalone/ReplMain.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+//      .map {
+//      case line if line.contains("docSearchPackages") =>
+////        val customDocSearchPackagesFullname = DocSearchPackages.getClass.getName.dropRight(1) // dropping `$` object name
+////        val customDocSearchPackagesFullname = getClass.getName.dropRight(1) // dropping `$` object name
+////        s"import _root_.$customDocSearchPackagesFullname"
+//        "TODO"
+//      case other => other
+//    }
+    ++ Seq(
+      s"import _root_.${getClass.getPackageName}.*"
```
Comment (DavidBakerEffendi) [inline src/main/scala/org/codeminers/standalone/ReplMain.scala:24]: @mpollmeier Would custom language extensions be brought into this REPL?

**Comment by mpollmeier** [comment]
i didn't clean up the PR, sorry, will follow up with a cleanup

File: src/main/scala/org/codeminers/standalone/ReplMain.scala (hunk 1 above)
Comment (mpollmeier) [reply src/main/scala/org/codeminers/standalone/ReplMain.scala:24]: you mean the custom steps etc? yes they are available, including the `.help` entries...
i added a few examples to the readme

Comment (DavidBakerEffendi) [reply src/main/scala/org/codeminers/standalone/ReplMain.scala:24]: Great!

**Pull request merged**